src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Application.cshtml

```diff
@@ -8,7 +8,18 @@
     Layout = "_Layout";
 }
 
-<a asp-action="InProgressApplications" asp-controller="RoatpGateway" class="govuk-back-link">Back</a>
+<div class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Dashboard">RoATP dashboard</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/@(Model.GatewayReviewStatus.Equals(GatewayReviewStatus.New) || Model.GatewayReviewStatus.Equals(GatewayReviewStatus.InProgress) ? Model.GatewayReviewStatus.Replace(" ", "") : "Closed")">RoATP applications</a>
```

Comment:
1. Should be using tag helper
2. Will blow up if Model.GatewayReviewStatus is null
3. Not doing safe case-insensitive equality